apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -324,12 +325,34 @@ class TopInstructionsCSP extends Component {
   };
 
   handleCommentTabClick = () => {
+    // Only increment visit count if user is switching from another tab to the
+    // comments tab.
+    if (this.state.tabSelected !== TabType.COMMENTS) {
+      this.incrementFeedbackVisitCount();
```

Comment:
What happens if I switch back and forth between the instructions and the feedback tab a couple times? Do we want to be counting that?

Reply:
i'm sorry, i missed this comment! yes, if a student switches back and forth between the two tabs, it would record multiple visits right now. we could set a buffer on how often we write to the metrics table (like, only record metrics every 5 minutes or something), but i'm not sure this is necessary in the initial implementation

Reply:
I think this is good enough to ship, since the number of unique users experiencing these events will still tell us lots. If we want to make this more accurate, we could wrap the event logger with [_.debounce](https://lodash.com/docs/4.17.11#debounce) (with the `leading` flag) to set a minimum interval between event logs. This wouldn't work across page loads, but could help guard against rapid tab switching.

Reply:
Sounds good just figured I would ask!

Reply:
nice! `debounce` sounds simple enough, so i'll go ahead and add it